### PR TITLE
Sdk license update

### PR DESCRIPTION
**Pull request by fatihkurtoglu**
We're changing the license from `MIT` to `Apache 2.0`. This PR will be published with the next version, as agreed. 

I'll add a note into `Release Notes` to announce this change.

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -1,21 +1,201 @@
...
+      To apply the Apache License to your work, attach the following
+      boilerplate notice, with the fields enclosed by brackets "[]"
+      replaced with your own identifying information. (Don't include
+      the brackets!)  The text should be enclosed in the appropriate
+      comment syntax for the file format. We also recommend that a
+      file or class name and description of purpose be included on the
+      same "printed page" as the copyright notice for easier
+      identification within third-party archives.
+
+   Copyright [yyyy] [name of copyright owner]
```
Comment (shaun-scale) [inline LICENSE:189]: @fatihkurtoglu - we'll want to update this part quick :)

**Review by shaun-scale — CHANGES_REQUESTED** [review]

File: LICENSE (hunk 1 above)
Comment (fatihkurtoglu) [reply LICENSE:189]: I was a bit confused on that part though (leaving as a template vs filling our details), and made some research and found that [discussion1](https://github.com/kubernetes/kubernetes-template-project/issues/3) and [discussion2](https://github.com/fluentassertions/fluentassertions/issues/1513) on k8 repo. That was the case for most enterprise's repos as well. Then left it as it is  :) Should we check again with the legal team?

Comment (shaun-scale) [reply LICENSE:189]: Ah damn - you did even more reading than I !! No, please just keep this as-is. 

Fatih, you are amazing with the details - thank you!

**Review by shaun-scale — APPROVED** [review]
LGTM

**Pull request merged**